src/app/reviews/reviews.component.scss

```diff
@@ -0,0 +1,25 @@
+
+.empty-status {
+  background: var(--ion-color-primary-contrast);
+  border-radius: 12px;
+  border: .5px solid var(--practera-card-border-color);
+  min-height: 58px !important;
+  h2 {
+    color: var(--practera-review-title);
+    font-size: 14px;
+  }
+}
+.top-round-btns {
+  .do {
+    --border-radius: 50px 0 0 50px;
+  }
+  .done {
+    --border-radius: 0 50px 50px 0;
+    margin-left: -4px;
+    
+  }  
+  .active-review {
+    --color: var(--ion-color-primary);;
```

Comment:
Use the directive `color="primary"` for this, do not create additional CSS

Reply:
we want to apply this color in a conditional situation, by getting this class. so it's not a permanent style, have to put in this special class.